### ISSUE TITLE
View of SQL script as strange characters

### ISSUE THREAD

**Issue by vksvicky** [TablePlus/TablePlus#2034]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): Not using any drivers for this activity

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 3.8.0 (333)

3. The steps to reproduce this issue: 
Open the basic.sql file using TablePlus, you can see as in the screenshot
Open the basic.sql file in any text editor, you can see the script

Noted: If the bug is related to data, please attach an example SQL data.
<img width="800" alt="Screenshot 2020-08-14 at 19 18 52" src="https://user-images.githubusercontent.com/1382081/90280898-94ee5f80-de63-11ea-9030-4d34ed4bfa3c.png">

[basic.sql.zip](https://github.com/TablePlus/TablePlus/files/5076377/basic.sql.zip)

**Comment by huyphams** [comment]
Hi @vksvicky it's reported here: https://github.com/TablePlus/TablePlus/issues/2022
TL;DR it's just a rendering issue, it will not affect the data you can wait for the next release or download the hotpatch here: 
https://www.dropbox.com/s/bskp6rf1julta52/TablePlus.dmg?dl=0
Thanks!
